### PR TITLE
Added title and description meta tags

### PR DESCRIPTION
**Pull request by askvortsov1**
Modified code to set the meta "title" attribute based on the current page's tag, or to "Tags" (translatable) for pages related to the tags extension.

### CODE REVIEW

**Comment by askvortsov1** [comment]
With the changes made in f1c65ae, this fully covers requirements of #69 and #48, which can probably both be closed if this is good to merge.

**Comment by luceos** [comment]
This is a very decent PR @askvortsov1 well done!

File: js/src/forum/addTagFilter.js [hunk 1]
```diff
@@ -44,8 +52,12 @@ export default function() {
...
   // Add a parameter for the IndexPage to pass on to the DiscussionList that
   // will let us filter discussions by tag.
-  extend(IndexPage.prototype, 'params', function(params) {
-    params.tags = m.route.param('tags');
+  extend(IndexPage.prototype, 'params', function (params) {
+    if (m.route.param('tags')) {
+      params.tags = m.route.param('tags');
+    } else {
+      params.tags = "";
+    }
```
Comment (franzliedke) [inline js/src/forum/addTagFilter.js]: Why is this change necessary exactly? Just making sure we have a string here instead of `undefined` if there's no tag? If so, do we want that, or do we want to remove the param?

File: js/src/forum/addTagFilter.js [hunk 2]
```diff
@@ -23,8 +23,16 @@ export default function() {
...
+  });
+
+  extend(IndexPage.prototype, 'config', function () {
+    const tag = this.currentTag();
+
     if (tag) {
-      vdom.attrs.className += ' IndexPage--tag'+tag.id();
+      app.setTitle(tag.name());
+    } else {
+      app.setTitle("");
```
Comment (franzliedke) [inline js/src/forum/addTagFilter.js]: Single quotes, please. :speak_no_evil: 

Sorry, we need a linter or a formatter. Will make that a priority.

Comment (franzliedke) [reply js/src/forum/addTagFilter.js]: Actually... does this mean that the title will suddenly be empty when moving from the unfiltered "All discussions" page to a tag page and then back to "All discussions"?

Comment (askvortsov1) [reply js/src/forum/addTagFilter.js]: Not sure why I had this, but after further testing it seems unnecessary.

File: js/src/forum/addTagFilter.js (hunk 1 above)
Comment (askvortsov1) [reply js/src/forum/addTagFilter.js]: As with above, don't remember why it was added, but seems to work fine without it so I removed it.

**Comment by askvortsov1** [comment]
Addressed

**Comment by franzliedke** [comment]
Thank you!

**Comment by askvortsov1** [comment]
@franzliedke the accompanying https://github.com/flarum/lang-english/pull/154 still needs to be merged

**Comment by franzliedke** [comment]
I know. I will change the keys, though.
